### ISSUE TITLE
Netbeans 11.3 out!

### ISSUE THREAD

**Issue by reddyshyam** [OpenBeans/OpenBeans#14]
Hi,

Any chance releasing openbeans for the same please? I love openbeans as it works out of the box. Thanks for the great share! 👍

**Comment by emilianbold** [comment]
Yes, I know :-) It's on my list. I've been also waiting on a macOS notarized AdoptOpenJDK release.

Out of curiosity -- which OS are you using?

**Comment by reddyshyam** [comment]
Hi Emilian,

Thanks for prompt reply. I am on Windows 10 Pro 64bit insider build 20H1 19041.113

Regards
Shyam

________________________________

**Comment by reddyshyam** [comment]
Hi @emilianbold,

Trust you are safe and well in these troubled times. 

Just checking if openbeans is kicking and happening as there is no activity for a long time now.

**Comment by emilianbold** [comment]
Thank you for checking in, it's nice to see some interest.

I would love to play with rebasing OpenBeans onto the latest NetBeans, but since society is basically shut down one has barely time for paid work, let alone open source.

You can consider OpenBeans development on a break until things are more normal.

**Comment by reddyshyam** [comment]
Hi @emilianbold,

Totally with you there! I am just glad that you are safe and sound! That is all matters. :)

**Comment by emilianbold** [comment]
But... why bother with 11.x when we can jump straight to 12.x!?

I guess December would be a good traditional time to release OpenBeans.

**Comment by Baibhab97** [comment]
Hi @emilianbold  I would like to take this up? What exactly needs to be done?

**Comment by emilianbold** [comment]
Oh, you're a courageous one! Well, we just need to:

1. rebase the 40 patches from
https://github.com/OpenBeans/OpenBeans/tree/master/pkgsrc-coolbeans/ide/patches
on
top of the NetBeans 12.1 branch
https://github.com/apache/netbeans/tree/release121
1.1 And most likely re-generate /
tweak patch-0001-Adds-OpenBeans-branding.patch in case branding looks off
2. Make pkgsrc happy by:
2.1 exporting the patches (and adding hashes to
https://github.com/OpenBeans/OpenBeans/blob/master/pkgsrc-coolbeans/ide/distinfo
)
2.2 add hashes to distinfo for the upstream releases
2.3 register the SITES. URL in
https://github.com/OpenBeans/OpenBeans/blob/master/pkgsrc-coolbeans/ide/Makefile
3. while you are at it pick up a newer JDK URL and register it in the
Makefile (
https://github.com/OpenBeans/OpenBeans/blob/master/pkgsrc-coolbeans/macos/Makefile
) and distinfo (
https://github.com/OpenBeans/OpenBeans/blob/master/pkgsrc-coolbeans/macos/distinfo
)

This can probably be done incrementally, by just starting with the bare
upstream and adding files / patches then seeing if anything breaks while
building.

--emi

**Comment by emilianbold** [comment]
PS: And git can actually help with the rebase for most. If you start
with the original branch release112u1 branch, then apply the existing
patches you can just `git rebase` onto the 121 branch...

--emi


--emi

**Comment by reddyshyam** [comment]
We will have NetBeans 12.2 in a week or so too.
